packages/inputs/package.json

```diff
@@ -20,5 +20,18 @@
   },
   "dependencies": {
     "@arkitema/brand": "^0.1.2"
+  },
+  "devDependencies": {
+    "@storybook/addon-essentials": "^7.0.22",
+    "@storybook/addon-interactions": "^7.0.22",
+    "@storybook/addon-links": "^7.0.22",
+    "@storybook/blocks": "^7.0.22",
+    "@storybook/react": "^7.0.22",
+    "@storybook/react-vite": "^7.0.22",
+    "@storybook/testing-library": "^0.0.14-next.2",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "storybook": "^7.0.22"
```

Comment:
Aren't these packages included in the root level package.json?

Reply:
oops yes mustve gotten added when creating the storybook